cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/CertificateRenewalTest.java

```diff
@@ -1078,22 +1058,21 @@ public void testExpiredCertsGetRemovedAuto(VertxTestContext context)
         assertThat(isCertInTrustStore(CA_CRT, initialClusterCaCertSecret.getData()), is(true));
 
         // add an expired certificate to the secret ...
-        initialClusterCaCertSecret.getData().put("ca-2018-07-01T09-00-00.crt",
-                Base64.getEncoder().encodeToString(
-                        TestUtils.readResource(getClass(), "cluster-ca.crt").getBytes(StandardCharsets.UTF_8)));
+        String clusterCert = Objects.requireNonNull(TestUtils.readResource(getClass(), "cluster-ca.crt"));
+        String encodedClusterCert = Base64.getEncoder().encodeToString(clusterCert.getBytes(StandardCharsets.UTF_8));
+        initialClusterCaCertSecret.getData().put("ca-2018-07-01T09-00-00.crt", encodedClusterCert);
+
```

Comment:
This code got too long to be easily readable with the null check, so broke it out.